rfcs/0049-flakes.md

```diff
@@ -137,12 +137,15 @@ A flake has the following attributes:
     not available for `github` repositories (see below), since they're
     fetched as tarballs rather than as Git repositories.
 
-  * `lastModified`: The commit time of the revision `rev`, in the
+  * `lastModifiedDate`: The commit time of the revision `rev`, in the
     format `%Y%m%d%H%M%S` (e.g. `20181231100934`). Unlike `revCount`,
     this is available for both Git and GitHub repositories, so it's
     useful for generating (hopefully) monotonically increasing version
     strings.
 
+  * `lastModified`: The commit time of the revision `rev` as an integer
```

Comment:
Needs to be `lastModifiedDate`

Reply:
No, `lastModified` is an integer (seconds in the epoch) while `lastModifiedDate` is a string representation.